src/Tempest/Http/src/Routing/Construction/RouteMatchingRegexesBuilder.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Routing\Construction;
+
+use Tempest\Http\Routing\Matching\MatchingRegexes;
+
+final readonly class RouteMatchingRegexesBuilder
```

Comment:
Would something like `RouteMatchingRegexBuilder` also be a correct name? `Regexes` feels weird to me, and I think `Regex` is already plural

Reply:
Makes sense, will rename

Reply:
done